www/css/homepage.css

```diff
@@ -159,6 +179,7 @@ input[type="checkbox"]{
 }
 
 .background{
+    display: none;
```

Comment:
I thought the spec says having a separate page for the pending surveys, but this seems to work fine too. See if @xuanrongzi has a problem with this. Otherwise it's good.